io/bufio.ts

```diff
@@ -212,7 +212,8 @@ export class BufReader implements Reader {
    * For simple uses, a Scanner may be more convenient.
    */
   async readString(_delim: string): Promise<string | Deno.EOF> {
```

Comment:
The leading `_` should be removed from this now... it was a signal to allow unused arguments in the signature.

Reply:
@kitsonk done, thanks.